internal/cmd/crc32_table/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+const (
+	disclaimer = `// Code generated by astits. DO NOT EDIT`
```

Comment:
Could you update the disclaimer to `// Code generated by astits using internal/cmd/crc32_table. DO NOT EDIT`